internal/backend/sessions/sessionworkflows/workflows.go

```diff
@@ -177,16 +193,12 @@ func (ws *workflows) sessionWorkflow(wctx workflow.Context, params *sessionWorkf
 	dwctx, done := workflow.NewDisconnectedContext(wctx)
 	defer done()
 
-	sessionDurationHistogram.Record(
-		metricsCtx,
-		duration.Milliseconds(),
-		metric.WithAttributes(
-			attribute.Bool("replay", isReplaying),
-			attribute.Bool("success", err == nil),
-		),
-	)
+	sessionDurationHistogram.Record(metricsCtx, duration.Milliseconds(),
+		metric.WithAttributes(attribute.Bool("replay", isReplaying), attribute.Bool("success", err == nil)))
+	sessionInvocationDelayHistogram.Record(metricsCtx, duration.Milliseconds(),
```

Comment:
you shouldn't log this metric if it's replay
it is only for the first invocation

Reply:
done